egs/librispeech/asr/simple_v1/espnet_utils/numericalizer.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 Xiaomi Corporation (Author: Guo Liyong)
+# Apache 2.0
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import Iterable, List, Optional, Union
+from pathlib import Path
+
+import numpy as np
+
+from torchtext.data.functional import load_sp_model
+
+
+class NumericalizerMixin(ABC):
+
+    def _assign_special_symbols(self):
+        # <sos> and <eos> share same index for model download from espnet model zoo
+        assert '<sos/eos>' in self.token2idx \
```

Comment:
At first sight, this class contains no data field at all; so the meaning of `self.token2idx`
is quite confusing here. Could you add more documentation here?